proposals/dao/tip_vebalotc.ts

```diff
@@ -32,23 +32,23 @@ const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: Named
   const vebalOtcHelper = await VeBalHelperFactory.deploy(
     vebalOtcBuyer, // _owner
     addresses.veBalDelegatorPCVDeposit, // _pcvDeposit
-    addresses.balancerGaugeStaker // _boostManager
+    addresses.balancerGaugeStakerProxy // _balancerGaugeStaker
   );
   await vebalOtcHelper.deployTransaction.wait();
   logging && console.log(`vebalOtcHelper: ${vebalOtcHelper.address}`);
 
-  // Deploy veBoostManager implementation
-  const VeBoostManagerFactory = await ethers.getContractFactory('VeBoostManager');
-  const veBoostManagerImplementation = await VeBoostManagerFactory.deploy(
+  // Deploy BalancerGaugeStakerV2 implementation
+  const balancerGaugeStakerV2Factory = await ethers.getContractFactory('BalancerGaugeStakerV2');
```

Comment:
Minor nit, for the contract factory we usually capitalise the first letter: `BalancerGaugeStakerV2Factory`